db/db.js

```diff
@@ -3,7 +3,7 @@
  * The mongodb server is configured in db/config.js
 */
 
-var User = require('./config').User;
+// var User = require('./config').User;
```

Comment:
hmm... looks different in my code. let me close this one out and submit a new pull request.